pkg/apps/util/util.go

```diff
@@ -512,8 +516,9 @@ func MakeDeploymentV1(config *deployapi.DeploymentConfig, codec runtime.Codec) (
 				deployapi.DesiredReplicasAnnotation:    strconv.Itoa(int(config.Spec.Replicas)),
 				deployapi.DeploymentReplicasAnnotation: strconv.Itoa(0),
 			},
-			Labels:          controllerLabels,
-			OwnerReferences: []metav1.OwnerReference{*controllerRef},
+			Labels:            controllerLabels,
+			OwnerReferences:   []metav1.OwnerReference{*controllerRef},
+			CreationTimestamp: defaultCreationTimestamp,
```

Comment:
@mfojtik why? we will now send objects to API server with CreationTimestamp set?

Reply:
huh, I didn't notice this is not just for test...